src/QMCDrivers/VMC/VMC.cpp

```diff
@@ -130,7 +130,8 @@ bool VMC::run()
   //copy back the random states
 #ifndef USE_FAKE_RNG
   for (int ip = 0; ip < NumThreads; ++ip)
-    *(RandomNumberControl::Children[ip]) = *(Rng[ip]);
+    if (Rng[ip])
```

Comment:
I don't think this if-test should be necessary.  If it is necessary, something is wrong with the code. Try replacing it with an assert.